source/Willow.package/WillowJavascriptNamespace.class/instance/setHandleAjaxErrorCallFunction..st

```diff
@@ -0,0 +1,4 @@
+Configuring
+setHandleAjaxErrorCallFunction: aJSFunction
```

Comment:
Rename method to one of the following:
a) setHandleAjaxErrorCallFunctionTo:
b) handleAjaxErrorsWith:
c) handleAjaxErrorsUsing: